### ISSUE TITLE
Bad code formatting in slides

### ISSUE THREAD

**Issue by coder006** [ipython/ipython#6350]
#### C
```c
int x = 29;
float y = 321.321;
double z = 32132132132133.21;
```
This as a markdown cell in a notebook renders into following in slides
```
C
<span class="dt">int</span> x = <span class="dv">29</span>;
<span class="dt">float</span> y = <span class="fl">321.321</span>;
<span class="dt">double</span> z = <span class="fl">32132132132133.21</span>;

```

**Comment by ellisonbg** [comment]
@damianavila have you fixed this one? Plans to work on it for 3.0? I will target to 4.0, but feel free to move it to 3.0 if you work on it.

**Comment by damianavila** [comment]
Will work on this today...
